config/theme-tabler.php

```diff
@@ -36,6 +36,12 @@
     // fluid
     // fluid_vertical
 
+    /**
+     * Pick a login page layout.
+     * Possible values: default, illustration, cover
+     */
+    'layout_login' => 'default', // default, illustration, cover
```

Comment:
I find it a bit backwards to call this `layout_login` instead of `login_layout`. I understand that it comes right after `layout` so it makes sense, but... I think `login_layout` is much easier to read in English and is worth the difference.

But speaking about layouts... I know I'm the one who said that only the `login` should follow these layouts buuuut.... when I use the `illustration` and `cover` layouts... then go to Recover Password or Register... those pages look too different...

So how about when you choose a `login_layout`, it also applies to the other 2-3 auth pages? And in that case we should probably call it:

```suggestion
    'auth_layout' => 'default', // default, illustration, cover
```